src/language/blockString.js

```diff
@@ -66,10 +66,14 @@ function leadingWhitespace(str) {
   return i;
 }
 
-function isBlank(str) {
+export function isBlank(str: string): boolean {
```

Comment:
No need to export this function for tests.
In tests, you can use new `trimStart` function.

In general, we never export functions to use them in tests, since tests should be independent of implementation.